extensions/adobe/experience/audiencemanager/segmentfolder.schema.json

```diff
@@ -0,0 +1,48 @@
+{
+  "meta:license": [
+    "Copyright 2018 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id": "https://ns.adobe.com/experience/audiencemanager/segmentfolder",
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "title": "Adobe Audience Manager Segment Folder Class",
+  "type": "object",
+  "description": "Adobe Audience Manager Segment Folder Class",
+  "meta:extensible": true,
+  "meta:abstract": true,
+  "meta:extends": ["https://ns.adobe.com/xdm/data/record"],
+  "definitions": {
+    "segmentfolder": {
+      "properties": {
+        "xdm:segmentFolderIdentity": {
+          "title": "Segment Identity",
```

Comment:
The title for this field is `Segement Identity` yet the property is named `Segment Folder Identity`. This seems to be a disconnect,.